### PR TITLE
Bug 1170150 Stop serving call-urls to Loop clients.

### PR DESCRIPTION
**Pull request by Standard8**
This stops us serving call-urls, and returns 405 instead, as discussed in the bug.

### CODE REVIEW

**Comment by ametaireau** [comment]
Tests failed because you also need to update the load tests.

File: test/functional_test.js [hunk 1]
```diff
@@ -417,7 +417,8 @@ function runOnPrefix(apiPrefix) {
         });
       });
 
-      it("should return a list of call-url with some information",
+      // Unable to test now that POST call-url is not available.
+      it.skip("should return a list of call-url with some information",
```
Comment (ametaireau) [inline test/functional_test.js]: We probably should update the functional tests to add the call-urls directly with a call to the database.

**Comment by Natim** [comment]
I am wondering if it is not just something we want to activate/deactivate with a configuration rule rather than removing all the code for it.

File: test/functional_test.js [hunk 2]
```diff
@@ -446,13 +451,16 @@ function runOnPrefix(apiPrefix) {
 
       it("should return a list of call-url with some information & subject",
```
Comment (Natim) [inline test/functional_test.js]: Why do we keep those tests if the feature don't exists anymore?

**Comment by Natim** [comment]
I order to fix the loadtest we could benefit from this: https://github.com/mozilla-services/readinglist/pull/232/files

File: test/functional_test.js (hunk 2 above)
Comment (Standard8) [inline test/functional_test.js]: Alexis asked for it to be retained: https://github.com/mozilla-services/loop-server/pull/337#discussion_r32159044

**Pull request merged**